recipes/smallgenomeutilities/0.3.8/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "smallgenomeutilities" %}
+{% set version = "0.3.8" %}
+{% set sha256 = "3b91f57061f1475b0d8770575fb24640dcded82c588b5fd9afbcc227801a494f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #url: https://pypi.io/packages/source/{{ name[0]|lower }}/{{ name|lower }}/{{ name|lower }}-{{ version }}.tar.gz
+  url: https://github.com/cbg-ethz/{{ name|lower }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -v
+
+requirements:
+  host:
+    - python >=3.6
```

Comment:
If you look at the Azure tests you can see your package is only built for python 3.10. You can build it for python 3.7, 3.8, 3.9 and 3.10 with these changes:

```
build:
  skip: True  # [py<37]

requirements:
  host:
    - python
  run:
    - python
```

by not specifying the python version in the requirements, all versions are build. Using the build skip, you then exclude the versions that don't work for you (with the comment).

Reply:
Ooops, we now get the error:
> ERROR: recipes/smallgenomeutilities/0.3.8/meta.yaml:16: should_not_be_noarch_skip: The recipe uses `skip: True` but is marked noarch

Reply:
From the [bioconda documentation](https://bioconda.github.io/contributor/linting.html#noarch-or-not-noarch):

> A noarch package cannot use skip. Packages that are noarch but require specific Python versions should use pinning (e.g. - python >3).

I think we should revert 0bf6bf5.

Reply:
My bad, sorry! I was reading from the [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/packages.html#noarch-python) docs, instead of the [bioconda](https://bioconda.github.io/developer/_autosummary/bioconda_utils.lint.check_noarch.html#bioconda_utils.lint.check_noarch.should_not_be_noarch_skip) docs...